### ISSUE TITLE
To many requests to fast error - API error

### ISSUE THREAD

**Issue by kaz911** [fjhorrillo/homebridge-airzone-cloud#26]
**Describe The Bug:**

Log files show lots of errors very fast after eachother. 
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded

**To Reproduce:**

just connected up the system to "New Airzone Cloud"
**Expected behavior:**

less requests per second? 

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:29] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:29] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 09:32:30] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 09:32:30] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:26] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:26] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:27] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:27] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:27] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:27] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded
[20/10/2022, 11:33:27] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[20/10/2022, 11:33:27] [AirzoneCloud] Error in getDeviceStatus. Error: API rate limit exceeded

**Plugin Config:**
default
```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by jessicadipalma** [comment]
Hi there,

I still have the same problem.

[25/08/2023, 19:16:34] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:34] [AirzoneCloud] Error in getDeviceStatus. Error
[25/08/2023, 19:16:34] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:34] [AirzoneCloud] Error in getDeviceStatus. Error
[25/08/2023, 19:16:34] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:34] [AirzoneCloud] Error in getDeviceStatus. Error
[25/08/2023, 19:16:34] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:34] [AirzoneCloud] Error in getDeviceStatus. Error
[25/08/2023, 19:16:34] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:34] [AirzoneCloud] Error in getDeviceStatus. Error
[25/08/2023, 19:16:39] [AirzoneCloud] Salon: Set Characteristic TargetTemperature -> 24.5 0
[25/08/2023, 19:16:39] [AirzoneCloud] Salon: Set Characteristic TargetTemperature -> 24.5ºC in 0.008s
[25/08/2023, 19:16:39] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:39] [AirzoneCloud] Error in setTemperature. Error
[25/08/2023, 19:16:39] [AirzoneCloud] Error calling to AirzoneCloud. Status: 429 Too Many Requests 
[25/08/2023, 19:16:39] [AirzoneCloud] Error in getDeviceStatus. Error

**Comment by fjhorrillo** [comment]
Fixed in v0.4.0
